wicket/src/update.rs

```diff
@@ -0,0 +1,241 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Types related to rack updates
+
+use semver::Version;
+use serde::{Deserialize, Serialize};
+use sha3::Digest;
+use sha3::Sha3_256;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::{Path, PathBuf};
+
+#[derive(Debug, Clone)]
+pub struct Sha3_256Digest([u8; 32]);
```

Comment:
I think this can be

```rust
#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct Sha3_256Digest(#[serde(with = "hex::serde")] [u8; 32]);
```

instead of manually implementing them.

Reply:
Ha! I knew there must have been an easier way. Thanks!